### PR TITLE
moocast gold

### PR DESCRIPTION
**Pull request by SansPapyrus683**
_If any of the below don't apply to this Pull Request, mark the checkbox as done._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions), which include the following: 
  - I understand that if it is clear that I have not attempted to follow these guidelines (ex. if I have not used tabs to indent), my PR will be closed.
  - If changes are requested, I will re-request the review after making the changes.

### CODE REVIEW

File: solutions/usaco-669.mdx [hunk 1]
```diff
@@ -267,76 +267,138 @@ print(last_dist, file=open('moocast.out', 'w'))
...
+	vector<pair<int, int>> cows(cow_num);
+	for (pair<int, int>& c : cows) {
+		read >> c.first >> c.second;
 	}
 
-	ll lo = 1;
-	ll hi = INT32_MAX;
-	ll sol = hi;
+	int lo = 0;
+	int hi = INT32_MAX / 2;  // should be large enough
```
Comment (ryanchou-dev) [inline solutions/usaco-669.mdx:319]: i think it's better if u just do `1e9` and write in the comment: "arbitrarily large integer" or smtg

cuz this number feels too specific and might get some ppl confused

File: solutions/usaco-669.mdx [hunk 2]
```diff
@@ -267,76 +267,138 @@ print(last_dist, file=open('moocast.out', 'w'))
...
+		int cowNum = Integer.parseInt(read.readLine());
+		int[][] cows = new int[cowNum][2];
+		for (int c = 0; c < cowNum; c++) {
+			StringTokenizer cow = new StringTokenizer(read.readLine());
+			cows[c][0] = Integer.parseInt(cow.nextToken());
+			cows[c][1] = Integer.parseInt(cow.nextToken());
+		}
+
+		int lo = 0;
+		int hi = Integer.MAX_VALUE / 2;  // should be large enough
```
Comment (ryanchou-dev) [inline solutions/usaco-669.mdx:354]: same here if u decide to apply

**Review by ryanchou-dev — APPROVED** [review]
yap yap

File: solutions/usaco-669.mdx (hunk 1 above)
Comment (SansPapyrus683) [reply solutions/usaco-669.mdx:319]: kk

**Review by dongliu0426 — APPROVED** [review]

**Pull request merged**